### PR TITLE
New schema fixes

### PR DESCRIPTION
**Pull request by dae**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt [hunk 1]
```diff
@@ -28,6 +28,8 @@ import android.util.Pair
 import androidx.annotation.CheckResult
 import androidx.annotation.VisibleForTesting
 import anki.search.SearchNode
+import anki.search.SearchNodeKt
+import anki.search.searchNode
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt:32]: duplicate import? Also, I assume we get the anki packages semi-auto from the Anki-Android-Backend? I mean, I'm 99% sure on that but my knowledge in this whole area is still thin so I'm figuring can't hurt to confirm

**Review by mikehardy — CHANGES_REQUESTED** [review]
LGreatTM - just a duplicate import

**Comment by mikehardy** [comment]
Also if you rebase on main the emulator tests should be non-flaky now, here's some android fun for you #12113 - in particular the root cause https://www.rainforestqa.com/blog/hunting-race-condition-in-android-10-emulator

**Review by david-allison — APPROVED** [review]
LGTM, the `postExecute` does the same as the `onProgress` in `CardBrowser` so it's fine to skip this

File: AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt:32]: One is a class, one is a method

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt:32]: Oh, where is the :facepalm: reaction when I need it? Thanks

**Review by mikehardy — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt (hunk 1 above)
Comment (dae) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Collection.kt:32]: > Also, I assume we get the anki packages semi-auto from the Anki-Android-Backend?

Yes, they're generated from .proto files in the desktop repo

**Pull request merged**